.github/dependabot.yml

```
@@ -27,3 +27,10 @@ updates:
         update-types: ['version-update:semver-major', 'version-update:semver-minor']
       # We are not ready to handle upper version of spectree, even for patches
       - dependency-name: 'spectree'
+
+  - package-ecosystem: 'github-actions'
+    directory: '.github/workflows'
+    schedule:
+      interval: 'daily'
+      time: '09:00'
```

```suggestion
      time: '06:00'
```
Pour libérer les runners le matin ?

---

Bonne remarque. Les pull requests pour les mises à jour de GitHub Actions vont déclencher des workflows assez court (aucun test ne sera exécuté), donc ça n'est pas très important à mon avis. Je vais quand même changer l'heure.

Par contre, je vais aussi changer l'heure pour les dépendances Python. Ainsi on aura 3 heures distinctes : 3h pour les paquets Python, 6h pour les GitHub Action et 9h pour les dépendances NPM.